libr/core/cmd_anal.c

```diff
@@ -101,6 +102,9 @@ static const char *help_msg_ae[] = {
 	"aeip", "", "initialize ESIL program counter to curseek",
 	"aek", " [query]", "perform sdb query on ESIL.info",
 	"aek-", "", "resets the ESIL.info sdb instance",
+	"aeli", "", "list loaded ESIL interrupts",
+	"aeli", " [file]", "load ESIL interrupts from shared object",
+	"aelir" " [interrupt number]", "remove ESIL interrupt and free it if needed",
```

Comment:
use `aeli-` instead

Reply:
i disagree, `aeli-` should in the future unload whole libraries, not specific interrupts

Reply:
(i see `cmd-` as undoing the effects of `cmd` in a way, so `aeli` would undo the loading of a file, therefore unload it, which aelir doesn't do, it just removes an interrupt)

Reply:
isnt this ael- vs aeli- ? imho the 'r' should not be used to remoev things because - is used in all the other commands.

Reply:
uhm yeah.. its a bit confusing, lets merge for now and ill find time to think better constructions here